app/assets/stylesheets/custom.scss

```diff
@@ -321,3 +321,25 @@ ul, menu, dir {
     width: 100px;
     height: 30px;
 }
+
+#flash_notice {
+  &.error {
+    border: solid 4px #900;
+          background-color: #FF6565;
+        }
+  &.notice {
+    border: solid 4px #26722D;
+          background-color: #A4E7A0;
```

Comment:
Color `#A4E7A0` should be written as `#a4e7a0`<br>Line should be indented 4 spaces, but was indented 10 spaces